### PR TITLE
Mattcrooke navbar2

### PR DESCRIPTION
**Pull request by matthewcrooke1**

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any styles
+ * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+ * file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+div#wrapper{
```
Comment (kevincox) [inline app/assets/stylesheets/application.css]: #wrapper is gone.

**Comment by kevincox** [comment]
I think this needs to be updated.

**Comment by ZachDFK** [comment]
looks good to me :+1:

**Comment by kevincox** [comment]
This is recreating files that have be renamed. Please don't merge.

**Pull request closed without merging**